openstudiocore/src/shared_gui_components/VariableList.cpp

```diff
@@ -380,7 +383,14 @@ void VariableListController::addItemForDroppedMeasureImpl(QDropEvent * event, bo
       // measure
       analysis::RubyMeasure measure(projectMeasure);
       try{
-        measure.setArguments(m_app->measureManager().getArguments(*project, projectMeasure));
+        auto model = m_app->currentModel();
+        boost::optional<Workspace> workspace;
+        if( model ) {
+          ProgressBar progress(m_app->mainWidget()); 
+          energyplus::ForwardTranslator translator;
+          workspace = translator.translateModel(model.get(),&progress);

```

Comment:
@macumber One thing that surprised me when I was looking through this is that Model Measure arguments are populated with correct model when viewing from App.  It had to be pulling them from the t_project, but I am surprised the project keeps a current model.